packages/dev/core/src/Lights/LTC/ltcTextureTool.ts

```diff
@@ -0,0 +1,109 @@
+import { RawTexture } from "core/Materials/Textures/rawTexture";
+import type { BaseTexture } from "core/Materials/Textures/baseTexture";
+import { Texture } from "core/Materials/Textures/texture";
+import type { Scene } from "core/scene";
+import { Constants } from "core/Engines/constants";
+import { Tools } from "core/Misc/tools";
+import type { Tuple } from "core/types";
+
+/**
+ * Interface that can provide LTC textures used by area Lights. Users can override the default ones to provide their own LTC textures.
+ */
+export interface IAreaLightLTCProvider {
+    /**
+     * Linearly trasnformed cossine texture for BRDF.
+     */
+    ltc1Texture: BaseTexture;
+
+    /**
+     * Linearly trasnformed cossine texture for BRDF.

```

Comment:
Is this intended to be the same comment as above? What's the difference between these two?